### PR TITLE
rules added for OCP-28297

### PR DESCRIPTION
**Pull request by yapei**
@yanpzhan  Can you help review?

### CODE REVIEW

File: features/step_definitions/resources.rb [hunk 1]
```diff
@@ -211,9 +211,13 @@
...
-    cb[cb_name] = clazz.list(user: _user, project: project(namespace))
+  if BushSlicer::ProjectResource > clazz
+    if in_project
+      cb[cb_name] = clazz.list(user: _user, project: project(namespace))
+    else
+      cb[cb_name] = clazz.list(user: _user, project: :all)
+    end
   else
-    cb[cb_name] = clazz.list(user: _user)
+    cb[cb_name] = clazz.list(user: _user)
```
Comment (liangxia) [inline features/step_definitions/resources.rb:221]: White spaces at end of line.

Comment (yapei) [reply features/step_definitions/resources.rb]: updated @liangxia

**Pull request merged**